homeassistant/components/sensor/metlink.py

```diff
@@ -0,0 +1,201 @@
+"""
+Support for Metlink's public transport.
+Bus, Train, Ferry. Wellington / Whanganui a Tara
+"""
+
+import logging
+from datetime import timedelta, datetime
+
+import requests
+
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ATTRIBUTION = "Data provided by metlink.org.nz"
+ICON = 'mdi:bus'
+STOP_URL = "https://www.metlink.org.nz/api/v1/StopDepartures/{stop_number}"
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Get the Metlink public transport sensor."""
+    # journal contains [0] Station ID start, [1] Station ID destination
+    # [2] Station name start, and [3] Station name destination
+    stop_number = config.get('stop_number')
+    route_number = config.get('route_number')
+    add_devices([MetlinkSensor(stop_number, route_number)])
+
+
+def pretty_timestamp(t):
```

Comment:
There are a lot of time/date related stuff available in `util/dt.py`. Can you please check if `pretty_timestamp()` is really needed?